CONTRIBUTING.md

```diff
@@ -102,7 +102,8 @@ black .
 ...Or install the pre-commit hooks: please refer to the
 [pre-commit](https://pre-commit.com/) documentation.
 
-Our code is also linted using `flake8`, with plugins `flake8-pyi` and `flake8-bugbear`. As with our other checks, running
+Our code is also linted using `flake8`, with plugins `flake8-pyi`
+`flake8-bugbear`, and flake8-noqa. As with our other checks, running
```

Comment:
```suggestion
`flake8-bugbear`, and `flake8-noqa`. As with our other checks, running
```

Reply:
Or maybe I was looking at an old version?

Reply:
Yeah, I already made this change!

Reply:
Weird that it's still showing "Commit suggestion" as an option